### ISSUE TITLE
DRep explorer, Dashboard - after Retire as DRep and becoming Direct Voter - user is presented like are Active DRep

### ISSUE THREAD

**Issue by pmbinapps** [IntersectMBO/govtool#1026]
To reproduce:

1. Become DRep
2. Delegate to Yourself
3. Become Direct Voter - on Dashboard user is shown as Direct Voter and Retired as Drep, and on DRep explorer Voting power is delegated to Active DRep  this user - the same way as he/she be still a DRep that delegated to him self.

NOTE: Screenshot below shows state just before the transaction retirement was updated to Retires state. State after transaction is completed is covered in this [issue](https://github.com/IntersectMBO/govtool/issues/1026)
![Image](https://github.com/IntersectMBO/govtool/assets/82152577/ac671e01-690f-43e6-b402-0b1e7bc40813)

**Comment by MSzalowski** [comment]
Let's keep it on hold until edge conditions between DRep and Direct Voter are clarified

**Comment by pmbinapps** [comment]
Similar aspect in this issue: https://github.com/IntersectMBO/govtool/issues/1010 - After actual Retirement was completed:


![Image](https://github.com/IntersectMBO/govtool/assets/82152577/f35aa942-6773-4ef1-8ff4-81542b9eec83)

**Comment by MSzalowski** [comment]
@bosko-m We can unblock this not displaying the delegation card in drep directory when the delegation is set for the direct voter

**Comment by bosko-m** [comment]
@m-i-k-e What do you think, is it acceptable? To me, it sounds fine.

**Comment by MSzalowski** [comment]
> @bosko-m We can unblock this not displaying the delegation card in drep directory when the delegation is set for the direct voter

Additional context:
Hide delegation on ourseleves, but display delegation for any other direct voter that user might have.

**Comment by m-i-k-e** [comment]
@m-i-k-e Check Figma

**Comment by pmbinapps** [comment]
Current status state after Retire as DRep - Retire as Direct Voter - Becoming Direct Voter:

OK on dashboard as in comment above
NOT OK - on DRep explorer still card with Delegation to Direct Voter visible:
![image](https://github.com/IntersectMBO/govtool/assets/82152577/cc951068-f458-4658-9cc8-78c0490ea0aa)
- this issue needs to be implemented https://github.com/IntersectMBO/govtool/issues/1010#issuecomment-2139305134

**Comment by pmbinapps** [comment]
OK NOW:



![Image](https://github.com/IntersectMBO/govtool/assets/82152577/ae3f00e3-dc29-4188-8e8e-183febaef256)
